en/config/actuators.md

```diff
@@ -55,50 +54,62 @@ Factors that cannot be configured are hard coded for the current airframe.
+The motor geometry explanations for a number of different frames are described below.
+
+
+#### Motor Position Coordinate System
+
+The coordinate system for motor positions is FRD (in body frame), where the X axis points forward, the Y axis to the right and the Z axis down.
+
+The **origin is the vehicle's centre-of-gravity (COG)**, and **NOT** the autopilot location).
+
+![Actuators CG reference diagram](../../assets/config/actuators/quadcopter_actuators_cg_reference.png)
 
 
 #### Motor Geometry: Multicopter
 
 The screenshot below shows the geometry screen for a multicopter frame with (right) and without (left) advanced settings.
 Multicopters don't have any control surfaces, so none are shown.
 
-![Geometry MC (QGC)](../../assets/config/actuators/qgc_actuators_mc_geometry.png)
+![Geometry MC (QGC)](../../assets/config/actuators/qgc_actuators_mc_geometry_marked.png)
 
-The **Motors** setting lets you choose the number of motors.
+First, the **Motors** drop-down setting lets you choose the number of motors, which is 4 for the example above.
 
-For each multicopter motor you can then set:
+For each motor you can then set:
 - `Position X`: [X-position](#motor-position-coordinate-system), in metres.
 - `Position Y`: [Y-position](#motor-position-coordinate-system), in metres.
 - `Position Z`: [Z-position](#motor-position-coordinate-system), in metres.
-- `Direction CCW`: Checkbox to indicate motor spins counter-clockwise (uncheck for clockwise).
-- `Bidirectional`: Checkbox to indicate motor is [bidirectional](#bidirectional-motors) 
-- `Slew Rate`: ?
+- (Advanced) `Direction CCW`: Checkbox to indicate motor spins counter-clockwise (uncheck for clockwise).
+- (Advanced) `Bidirectional`: Checkbox to indicate motor is [bidirectional](#bidirectional-motors) 
+- (Advanced) `Slew Rate`: ?
+
+The X, Y, Z positions are in [FRD coordinate frame, relative to the centre of gravity](#motor-position-coordinate-system).
 
-The X, Y, Z positions are in [FRD relative to the centre of gravity](#motor-position-coordinate-system).
 
+#### Motor Geometry: VTOL Quadrotor Tailsitter
 
-#### Motor Geometry: VTOL Tailsitter
+VTOL Quadrotor is a tailsitter with 4 motors like a quadcopter as shown below.
 
-The default tailsitter motor geometry is shown below.
-Motors have the same configuration fields as for the [multicopter geometry](#motor-geometry-multicopter).
+![VTOL Quadrotor Tailsitter](../../assets/airframes/types/VTOLQuadRotorTailSitter.svg)
+![VTOL Quadrotor Tailsitter Real Example](../../assets/airframes/vtol/skypull/skypull_sp1.jpg)
+
+The default motor geometry is shown below. Motors have the same configuration fields as for the [multicopter geometry](#motor-geometry-multicopter).
 
 ![Geometry motor: tailsitter vtol](../../assets/config/actuators/qgc_geometry_tailsitter_motors.png)
 
 #### Motor Geometry: VTOL Tiltrotor
 
 The default VTOL tiltrotor motor geometry is shown below.
 
+![Tiltrotor VTOL Airframe](../../assets/airframes/types/VTOLTiltRotor.svg)
```

Comment:
That image is a hexarotor tiltrotor. 

It is just too big. Ditto with the quad tailstitter. 

FYI only. My point - worth rendering this to see what it looks like. To be clear, I think having an image is a good idea, but one that pushes the content a page down is annoying for readers.

I'll see if I can constrain these.

Reply:
I agree. The Airfarme SVGs are quite confusing! Will create PR for that as well

Reply:
PR added here: https://github.com/PX4/PX4-Autopilot/pull/19686